celery/backends/couchdb.py

```diff
@@ -91,13 +93,16 @@ def get(self, key):
             return None
 
     def set(self, key, value):
-        data = {'_id': key, 'value': value}
+        # data = {'_id': key, 'value': value}
+        data = json.loads(value)
```

Comment:
Why is this necessary? I feel if the couchdb needs to inject anything into the generated structure it should do so before the structure is serialized, e.g. by overriding `encode_result()` or `encode()`.